research-software-directory/RSD-as-a-service
# Add list of projects for the user

# Add list of projects for the user

Fixes #283

Changes proposed in this pull request:

* add an overview of all projects to the user page for which the user is the maintainer of
* fix some minor logging messages
* remove an unused file (accidentally added some time ago?)

How to test:

* add some projects with a logged in user and you should see them in the projects list on the user page
* adding projects with another user should only display the projects for which the user is maintainer of

PR Checklist:
